File: PHP/uberBlack.php
<?php 
require_once('car.php');
class UberBlack extends Car {
    private $typeCarAccepted;
    private $seatMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatMaterial) {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatMaterial = $seatMaterial;
    }
}

?>

File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    #Atributos de la clase
    typeCarAccepted = []
    seatMaterial = []

    #Constructor
    def __init__(self, license, driver, typeCarAccepted, seatMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatMaterial = seatMaterial